### PR TITLE
Add canonical icon

### PR DESCRIPTION
**Pull request by hiafi**
## TODO

- [x] Documentation

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit e5e1ba83c53f74bb103d6dc05474cf1ed4c98a66

https://deploy-preview-427--gestalt.netlify.com

File: packages/gestalt/src/icons/canonical-pin.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M19 20c0 .551-.448 1-1 1h-7.471c-.551 0-1-.449-1-1v-2H14a2 2 0 0 0 2-2V8h2c.552 0 1 .449 1 1v11zM5.705 9.996a1.126 1.126 0 0 1 1.59-1.591L8.7 9.809l3.004-3.004a1.125 1.125 0 1 1 1.592 1.591L8.7 12.991 5.705 9.996zM18 6h-2V3a2 2 0 0 0-2-2H5a2 2 0 0 0-2 2v13a2 2 0 0 0 2 2h2.529v2c0 1.654 1.346 3 3 3H18c1.654 0 3-1.346 3-3V9c0-1.654-1.346-3-3-3z" fill="#8E8E8E" fill-rule="evenodd"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/canonical-pin.svg]: This does not look like an optimized SVG path, PDS usually runs these through https://github.com/svg/svgo to minimize. Can we check to make sure it’s approved?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,11 @@
 
 ### Minor
 
+- Icon: adding a new icon for canonical pins
+
+### Minor
```
Comment (pfarejowicz) [inline CHANGELOG.md]: This looks like a bad merge

**Comment by hiafi** [comment]
I just got a new computer, and I am not sure how to change the branch this PR is based off of, so I am going to make a different one

**Pull request closed without merging**